java-strings-ops-2/src/main/java/com/baeldung/string/Anagram.java

```diff
@@ -0,0 +1,50 @@
+package com.baeldung.string;
+
+import java.util.Arrays;
+import com.google.common.collect.HashMultiset;
+import com.google.common.collect.Multiset;
+
+public class Anagram {
+    public boolean isAnagramSort(String str1, String str2) {
```

Comment:
Rather than `str1` and `str2` I think we can afford `string1` and `string2` - it's less cryptic and more friendly. Let's do that throughout the file.